developer-docs-site/docs/guides/system-integrators-guide.md

```diff
@@ -61,135 +61,6 @@ Aptos currently provides three SDKs:
 * [REST API spec](https://fullnode.devnet.aptoslabs.com/v1/spec#/)
 * [Local testnet development flow](/guides/local-testnet-dev-flow)
 
-## Accounts on Aptos
```

Comment:
Consolidated under Concepts > Accounts (in a different PR that's already been merged)